pkg/remote/trans/default_server_handler.go

```diff
@@ -333,3 +353,11 @@ func getRemoteInfo(ri rpcinfo.RPCInfo, conn net.Conn) (string, net.Addr) {
 	}
 	return ri.From().ServiceName(), rAddr
 }
+
+func (t *svrTransHandler) getDefaultSvcInfo() (svcInfo *serviceinfo.ServiceInfo) {
+	for _, svc := range t.svcMap {
```

Comment:
every time, for range will create a map iterator, and it hurts performance, maybe you should cache the default svcInfo by wrap map[string]*serviceinfo.Service with 
```golang
type Services struct {
	svcs map[string]*serviceinfo.Service
	default *serviceinfo.Service
}

func (*Services) Set(k, v) {}

func (*Services) Get(k) {}

func (*Services) GetDefault() {}

```